Packages/Netherlands3D/T3DPipeline/Runtime/Scripts/ParseCityJSON/CityJSONStructure/CityJSON.cs

```diff
@@ -17,19 +17,19 @@ public class CityJSON : MonoBehaviour
         //nodes defined in a CityJSON and therefore reserved node keys.
         private static string[] definedNodes = { "type", "version", "CityObjects", "vertices", "extensions", "metadata", "transform", "appearance", "geometry-templates" };
 
-        public string Version { get; private set; }
+        public string Version { get; private set; } = "1.0"; //default version
         public JSONNode Extensions { get; private set; }
         public JSONNode Metadata { get; private set; }
         public Vector3Double TransformScale { get; private set; } = new Vector3Double(1d, 1d, 1d);
         public Vector3Double TransformTranslate { get; private set; } = new Vector3Double(0d, 0d, 0d);
         public JSONNode Appearance { get; private set; }
         public JSONNode GeometryTemplates { get; private set; }
 
-        public CityObject[] CityObjects { get; private set; } = new CityObject[0];
+        public List<CityObject> CityObjects { get; private set; } = new List<CityObject>();
         public Vector3Double MinExtent { get; private set; }
         public Vector3Double MaxExtent { get; private set; }
         public Vector3Double AbsoluteCenter { get { return (MaxExtent + MinExtent) / 2; } }
-        public CoordinateSystem CoordinateSystem { get; private set; }
+        public CoordinateSystem CoordinateSystem { get; private set; } = CoordinateSystem.Unity;
```

Comment:
Default coordinate system is niet WGS84?

Reply:
nee, het is een generieke CityJSON lezer, dus coordinaten moeten alleen geconverteerd worden als er een expliciet coordinatensysteem is gespecificeerd